### PR TITLE
Add new localized strings on main page & settings page, add pl translation of new localized strings

### PR DESCRIPTION
**Pull request by wojtekmaj**

### CODE REVIEW

**Comment by wojtekmaj** [comment]
Wait, I need to add strings to all languages even if I can't translate them? 🤔 Why? 🤔

**Comment by wojtekmaj** [comment]
Wait, I need to add new English strings to all locales I don't know? 🤔

**Comment by Gusted** [comment]
#3091 I already made a begin of this. and all the other languages need the translations as well in order to work...

File: src/ui/popup/main-page/app-switch.tsx [hunk 1]
```diff
@@ -42,15 +42,13 @@ export default function AppSwitch(props: ViewProps) {
...
-        'Extension is enabled' :
-        isOff ?
-            'Extension is disabled' :
-            isTimeAutomation ?
-                'Switches according to specified time' :
-                isLocationAutomation ?
-                    'Switched according to location' :
-                    'Switches according to system dark mode';
+    const descriptionText = (() => {
+        if (isOn) return getLocalMessage('extension_enabled');
```
Comment (Gusted) [inline src/ui/popup/main-page/app-switch.tsx:46]: No `{ }`? .... Run `npm run code-style`

File: src/ui/popup/main-page/app-switch.tsx [hunk 2]
```diff
@@ -42,15 +42,13 @@ export default function AppSwitch(props: ViewProps) {
...
-    const descriptionText = isOn ?
-        'Extension is enabled' :
-        isOff ?
-            'Extension is disabled' :
-            isTimeAutomation ?
-                'Switches according to specified time' :
-                isLocationAutomation ?
-                    'Switched according to location' :
-                    'Switches according to system dark mode';
+    const descriptionText = (() => {
```
Comment (Gusted) [inline src/ui/popup/main-page/app-switch.tsx:45]: `(() =>`
A new closure to get the variable can cause memory leaks as this is called a lot due it's the main page for older version of browsers(Bugs that are already fixed but compatability) so let's use the old `? : ? :`

File: src/ui/popup/main-page/index.tsx [hunk 3]
```diff
@@ -5,6 +5,7 @@ import AppSwitch from './app-switch';
 import HelpGroup from './help';
 import SiteToggleGroup from './site-toggle';
 import ThemeGroup from './theme-group';
+import {getLocalMessage} from 'utils/locales';
```
Comment (Gusted) [inline src/ui/popup/main-page/index.tsx:8]: `rollup-plugin-typescript2` The compiler for typescript request exact path for imports so this should become something like `import {getLocalMessage} from '../../../utils/locales';`

File: src/ui/popup/main-page/app-switch.tsx (hunk 1 above)
Comment (wojtekmaj) [reply src/ui/popup/main-page/app-switch.tsx:46]: I have ESLint working :) No errors here!

File: src/ui/popup/main-page/app-switch.tsx (hunk 2 above)
Comment (wojtekmaj) [reply src/ui/popup/main-page/app-switch.tsx:45]: 👍

File: src/ui/popup/main-page/index.tsx (hunk 3 above)
Comment (wojtekmaj) [reply src/ui/popup/main-page/index.tsx:8]: 👍

File: src/ui/popup/main-page/app-switch.tsx (hunk 1 above)
Comment (Gusted) [reply src/ui/popup/main-page/app-switch.tsx]: Oh weird...

**Comment by plainheart** [comment]
Hi,
How's this PR for translation improvement going? I was going to add untranslated fields but just found this PR has been doing it.
If possible, I hope this work may go ahead so that more other languages can keep updated.
Thank you all!

**Comment by Gusted** [comment]
Hi @plainheart!

I think after the new Design is polished and ready for production, all hard-coded strings will be converted to localized strings and everyone can add the translation for their language.

Regards,
Gusted

**Pull request closed without merging**